www/themes/Charisma/templates/basepage.tpl

```diff
@@ -210,6 +210,16 @@
 <!-- tinymce editor -->
 <script type="text/javascript"
 		src="{$smarty.const.WWW_THEMES}/shared/libs/tinymce-builded/js/tinymce/tinymce.min.js"></script>
+<!-- library for iphone style toggles -->
+<script type="text/javascript" 
+		src="{$smarty.const.WWW_THEMES}/{$theme}/js/jquery.iphone.toggle.js"></script>
+<!-- jQuery RATY Plugin -->
+<script type="text/javascript"
+                src="{$smarty.const.WWW_THEMES}/{$theme}/js/jquery.raty.js"></script>
```

Comment:
What is the purpose of these?

Reply:
When using the charisma theme, there are always 404/console errors for these missing .js files

Also 1 addition to allow woff2 fonts to be loaded as well otherwise .htaccess has it blocked.

As far as being used by the theme I haven't gotten that far yet, just trying to fix the errors

Reply:
Thanks for the clarification,

Reply:
np, just reinstalled and waiting for pp to start working before I start adding my changes...

Here are the console errors I have noted with those these added
![image](https://user-images.githubusercontent.com/21246660/67603150-1a264d80-f746-11e9-91e0-083aa5ecaa03.png)

Reply:
I can go ahead and cancel this and reopen when I have all of charisma completed, that way there is no "incomplete" code/patches.

I mean its been this long without complaint so cant be that important, the only one I think is pressing is maybe the woff2 font in .htaccess but then again its been awhile with no complaint so I think it may be fine to wait